src/biokbase/narrative/__init__.py

```diff
@@ -13,22 +13,4 @@ def version():
 #   no args = print current version
 #   1 arg = self-modify version to arg value
 if __name__ == "__main__":
-    import os
```

Comment:
This is some fancy self-updating thing that never gets used. We might consider a helper script that helps out releases. But not yet.